example/v1/example.pb.dynamo.go

```diff
@@ -104,7 +105,19 @@ func (p *User) MarshalDynamoDBAttributeValue() (types.AttributeValue, error) {
 	var err error
 	sb.Reset()
 	_, _ = sb.WriteString("examplepb_v1_user:")
+	if len(p.GetId()) == 0 {
+		panic(fmt.Sprintf("sharded key: sort key field '%s' cannot be empty", "id"))
+	}
+	var pkskBuilder strings.Builder
```

Comment:
why can't this reuse the local buffer? of `sb`?

Reply:
It can and now it does